### DISCUSSION TITLE
Welcome to rkd_package Discussions!

### DISCUSSION THREAD

**Discussion by Conorisco** [PARADISE-AAV/rkd_package#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MACTCD68** [comment]
@Conorisco and other 
Just if we think about a task to do, we just add in the project with an issue. I create few new labels to distinguish the different task.

**Reply by MACTCD68** [reply]
@Conorisco I update the README that the pipeline is easiest to find

**Reply by rcheva** [reply]
Hi, I am Julio, I am nephrologist and work in Tallaght and Tullamore.
I have some experience in clinical research, statistics, known SPSS, enthusiast with R actually migrating from SPSS. 
I am here mainly to learn, although I will be delighted to collaborate
